data/json/items/comestibles.json

```diff
@@ -20468,5 +20468,62 @@
             "fruit": "mushroom_poison",
             "grow" : 91
         }
-}
+},{
+        "type" : "COMESTIBLE",
+        "id" : "eggnog",
+        "name" : "eggnog",
+        "weight" : 258,
+        "color" : "white",
+        "addiction_type" : "none",
+        "spoils_in" : 4,
+        "stim" : 0,
+        "container" : "jug_plastic",
+        "to_hit" : 0,
+        "comestible_type" : "DRINK",
+        "symbol" : "~",
+        "quench" : 25,
+        "healthy": 1,
+        "addiction_potential" : 0,
+        "nutrition" : 10,
+        "description" : "Smooth and rich, this spoon-coating mix of milk, cream, and eggs is a popular traditional holiday drink. While often spiked, it is still delicious on its own.  Meant to be stored cold, it will spoil rapidly.",
+        "price" : 40,
+        "material" : ["milk", "egg"],
+        "tool" : "null",
+        "volume" : 1,
+        "cutting" : 0,
+        "phase" : "liquid",
+        "charges" : 1,
+        "bashing" : 0,
+        "flags" : ["EATEN_COLD", "EATEN_HOT"],
+        "fun" : 5
+    },{
+        "type" : "COMESTIBLE",
+        "id" : "eggnog_spiked",
+        "name" : "spiked eggnog",
+        "weight" : 258,
+        "color" : "white",
+        "addiction_type" : "alcohol",
+        "spoils_in" : 0,
+        "use_action" : "ALCOHOL",
+        "stim" : -10,
+        "container" : "jug_plastic",
+        "to_hit" : 0,
+        "comestible_type" : "DRINK",
+        "symbol" : "~",
+        "quench" : 25,
+        "healthy" : -1,
+        "addiction_potential" : 5,
+        "nutrition" : 8,
+        "description" : "Smooth and rich, this spoon-coating mixture of milk, cream, eggs, and booze is a popular traditional holiday drink.  Having been fortified with alcohol, it will keep for a long time.",
+        "price" : 45, "//" : "A 12-charge gallon currently goes for around US$4, and eggnog is a bit more expensive than that.",
+        "material" : ["milk", "alcohol"],
+        "tool" : "null",
+        "volume" : 1,
+        "cutting" : 0,
+        "phase" : "liquid",
+        "charges" : 1,
+        "bashing" : 0,
+        "flags" : ["EATEN_COLD","EATEN_HOT"],
```

Comment:
I think those two flags are currently mutually exclusive.
The heating code doesn't check whether the item is cold and the cooling code doesn't check whether its hot. From what I can tell, this would make it possible for eggnog to be both hot and cold at the same time.